### PR TITLE
add(designation): software engineer

### PR DESCRIPTION
**Pull request by vishnuitta**
This consists of expectations from software engineer, sr software engineer, lead engineer irrespective of the kind of role (like frontend/backend).

Signed-off-by: Vitta <vitta@mayadata.io>

### CODE REVIEW

File: designation/software-engineer.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Software Engineer
```
Comment (kmova) [inline designation/software-engineer.md:1]: Looks good @vishnuitta 

Can you push this under job-roles directory and it is good to merge.

Comment (vishnuitta) [reply designation/software-engineer.md]: changed the directory Kiran.

File: job-roles/software-engineer.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+- Provide timely help to customers
+- Trainings to support team with required information
+- Providing information for documentation team
+- MOM of the communication
+- Helping other related opensource users
+
+#### Social media engagement
+This is about building self technical image on social media which in-turn increases that of company.
+- Completing surveys
+- Adovacy of component you worked in other mediums like K8s channels, or other opensource channels
```
Comment (qiell) [inline job-roles/software-engineer.md:53]:
```suggestion
- Advocacy of component you worked in other mediums like K8s channels, or other opensource channels
```

File: job-roles/sr-software-engineer.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+- Conduct training to support team with the help of team
+- Provide information to documentation team at project/repo level
+- Provide timely help to customers
+- RCA of user/customer issues
+- Being active and well known in other related opensource projects
+
+#### Social media engagement
+This is about building self technical image on social media which in-turn increases that of company.
+- Completing surveys
+- Strong adovacy of component you worked in other mediums like K8s channels, or other opensource channels
```
Comment (qiell) [inline job-roles/sr-software-engineer.md:52]:
```suggestion
- Strong advocacy of component you worked in other mediums like K8s channels, or other opensource channels
```

File: job-roles/lead-engineer.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+- Make sure required trainings happened with support team
+- Make sure dev/E2E teams have enough time in their tasks by empowering support team in solving user/customer issues
+- Avoid any escalations from customers with timely and apt response
+- Review RCAs and improve its format to convert them to product backlog
+- Being active and leading few interested feature areas in other related opensource projects
+
+#### Social media engagement
+This is about building self technical image on social media which in-turn increases that of company.
+- Completing surveys
+- Strong adovacy of component you worked in other mediums like K8s channels, or other opersource channels
```
Comment (qiell) [inline job-roles/lead-engineer.md:51]:
```suggestion
- Strong advocacy of component you worked in other mediums like K8s channels, or other opensource channels
```

File: job-roles/lead-engineer.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
+# Lead Engineer
+
+## Overview
+
+At MayaData we believe that the best way to deliver storage and related services to containerized and cloud-native environments is with containerized and cloud-native architectures.  Everything we do is focused on providing data agility and simplifying the daily operation of Kubernetes SRE’s and DevOps teams using Kubernetes as a data layer.  We enable our users to handle complex data management operations easily, and we have become open source leaders in building Kubernetes into a first-class data platform.  
+
+Lead engineers are the first level full time engineers at MayaData with exceptional knowledge on product and technologies used. This provides the change for individual to grow from team level to across multiple teams and company. Below are the expectations from them in-general irrespective of the kind of job role. They are also expected to fill the [sr software engineers](sr-software-engineer.md) related ones.
+
+### Expectations
```
Comment (bobek) [inline job-roles/lead-engineer.md:9]: Can we push it by one level? E.g.

```
## Expectations
### Process management
```

Thank you

Comment (vishnuitta) [reply job-roles/lead-engineer.md]: Moved one level up and repushed changes after rebase

**Comment by bobek** [comment]
@vishnuitta please include also update to `SUMMARY.md` that is used to build the https://mayadata-io.github.io/culture/ . Also I am not sure, how this plays with other texts we have already there. For example 
```
- [Backend Engineer](job-roles/backend-engineering.md)
- [Frontend Engineer](job-roles/frontend-engineering.md)
- [Solution Engineer](job-roles/solution-engineer.md)
```

This work seems to be almost like orthogonal to the actual job-descriptions. Maybe we want to merge them into on page describing different levels of jobs? Or maybe I am missing the "why". Thank you!

**Comment by vishnuitta** [comment]
> @vishnuitta please include also update to `SUMMARY.md` that is used to build the https://mayadata-io.github.io/culture/ . Also I am not sure, how this plays with other texts we have already there. For example
> 
> ```
> - [Backend Engineer](job-roles/backend-engineering.md)
> - [Frontend Engineer](job-roles/frontend-engineering.md)
> - [Solution Engineer](job-roles/solution-engineer.md)
> ```
> 
> This work seems to be almost like orthogonal to the actual job-descriptions. Maybe we want to merge them into on page describing different levels of jobs? Or maybe I am missing the "why". Thank you!

Updated Summary.md file. And, yes, it seems to be orthogonal.

**Review by kmova — APPROVED** [review]

**Review by bobek — APPROVED** [review]

**Pull request merged**